DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: crayon
 Title: Colored Terminal Output
-Version: 1.3.1
+Version: 1.3.2
 Authors@R: person("Gabor", "Csardi", , "csardi.gabor@gmail.com", role = c("aut", "cre"))
```

Comment:
Can you please add yourself to the authors, and also a bullet point to the NEWS? Thanks.